test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/DeferWithLabelTestSnapshot.kt

```diff
@@ -0,0 +1,102 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.defer
+
+import graphql.nadel.tests.next.ExpectedNadelResponse
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots
+ */
+@Suppress("unused")
+public class DeferWithLabelTestSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            ExpectedServiceCall(
+                service = "defer",
+                query = """
+                | {
+                |   defer {
+                |     hello
+                |     ... @defer(label: "slow-defer") {
+                |       slow
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                variables = "{}",
+                response = """
+                | {
+                |   "defer": {
+                |     "hello": "helloString"
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResponses = listOfJsonStrings(
+                    """
+                    | {
+                    |   "hasNext": false,
+                    |   "incremental": [
+                    |     {
+                    |       "path": [
+                    |         "defer"
+                    |       ],
+                    |       "label": "slow-defer",
+                    |       "data": {
+                    |         "slow": "slowString"
+                    |       }
+                    |     }
+                    |   ]
+                    | }
+                    """.trimMargin(),
+                ),
+            ),
+        )
+
+    /**
+     * ```json
+     * {
+     *   "data": {
+     *     "defer": {
+     *       "slow": "slowString"
+     *     }
+     *   }
+     * }
+     * ```
+     */
+    override val response: ExpectedNadelResponse = ExpectedNadelResponse(
+            response = """
+            | {
+            |   "data": {
+            |     "defer": {
+            |       "hello": "helloString"
+            |     }
+            |   },
+            |   "hasNext": true
```

Comment:
Not sure why `hasNext` is present on the final expected response, but not on the service response.